webapp/src/app.tsx

```diff
@@ -501,7 +502,7 @@ export class ProjectView
                 switch (t.subtype) {
                     case 'steploaded':
                         let tt = msg as pxsim.TutorialStepLoadedMessage;
-                        let showCategories = tt.showCategories ? tt.showCategories : Object.keys(tt.data).length > 7;
+                        let showCategories = (tt.showCategories ? tt.showCategories : Object.keys(tt.data).length > 7) ? ShowCategoryMode.All : ShowCategoryMode.None;
                         this.editor.filterToolbox({ blocks: tt.data, defaultState: pxt.editor.FilterState.Hidden }, showCategories);
```

Comment:
Is this right? Setting showcateogrymode to all for tutorial mode? Should probably be set to basic here